docs/knowledgebase/integrate/encode-txs.md

```diff
@@ -0,0 +1,58 @@
+---
+title: Encoding Transactions
+---
+
+The purpose of this document is to describe the specification for encoding
+[transactions](../../knowledgebase/learn-substrate/extrinsics) to submit to Substrate-based chains.
+
+## Dispatchable Calls
+
+Transactions on Substrate-based chains are used to invoke
+"[dispatchable calls](../../knowledgebase/getting-started/glossary#dispatch)", which are encoded as
+follows:
+
+- 1 byte: `04`, magic number that identifies a dispatchable call
+- 1 byte: 0-based module index from
+  [`construct_runtime!`](../../knowledgebase/runtime/macros#construct_runtime) (modules without
+  calls are ignored)
+- 1 byte: 0-based call index that corresponds to the target variant on the module's `Call` enum
+- variable: [SCALE](../../knowledgebase/advanced/codec)-encoded function parameters
+- 32 bytes: [address](../../knowledgebase/learn-substrate/account-abstractions) of function caller
+
+## Signed Extensions
+
+Substrate uses
+"[signed extensions](../../knowledgebase/learn-substrate/extrinsics#signed-extension)" to allow
+developers to extend its transaction-verification capabilities. Despite the name, signed extensions
+can also be applied to unsiqned transactions. They are intended to be used in any situation where it
```

Comment:
```suggestion
can also be applied to unsigned transactions. They are intended to be used in any situation where it
```

Reply:
Stop trying to extinquish my creativity, Joe :expressionless: